release/nightly_tests/shuffle_data_loader/shuffle_data_loader_test.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Exit immediately if any command fails.
+set -exo pipefail
+
+data_dir="s3://shuffling-data-loader-benchmarks/data/"
+stats_dir="s3://shuffling-data-loader-benchmarks/stats/"
+
+num_rows=$((4 * (10 ** 8)))
+num_row_groups_per_file=5
+batch_size=250000
+num_trials=2
+num_epochs=10
+
+max_concurrent_epochs_list=(2)
+num_files_list=(25)
+num_trainers_list=(4)
+num_reducers_per_trainer_list=(2)
+
+for max_concurrent_epochs in "${max_concurrent_epochs_list[@]}"; do
+        for num_files in "${num_files_list[@]}"; do
+                for num_trainers in "${num_trainers_list[@]}"; do
+                        for num_reducers_per_trainer in "${num_reducers_per_trainer_list[@]}"; do
+                                num_reducers=$(( num_reducers_per_trainer * num_trainers ))
+                                python ~/ray_shuffling_data_loader-feat-benchmarks/benchmarks/benchmark.py \
```

Comment:
This will need to be changed to `ray submit /path/to/cluster/config.yaml /path/to/benchmark.py` or `ray exec /path/to/cluster/config.yaml "python /path/to/mounted/dir/benchmark.py"` when running on a multi-node cluster.